starter-code/index.html

```diff
@@ -3,40 +3,85 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <div class="container">
+  <nav class="nav-bar nav-view">
+    <img src="./images/spotify-logo.png" class="logo" alt="Spotify Logo">
+    <ul class="links">
+      <li><a href="#">Premium</a></li>
+      <li><a href="#">Discover</a></li>
+      <li><a href="#">Help</a></li>
+      <li><a href="#">Download</a></li>
+    </ul>
+  </nav>
+
+  <div class="landing view">
+    <h1>Music for everyone.</h1>
+    <h3>Spotify is now free on mobile, tablet and computer.<br/>
+    Listen to the right music, wherever you are.</h3>
+  </div>
+
+  <div class="height about-height">
```

Comment:
There is an other way to write this kind of code, you can use the list tag :) It's the same at the end, but it may be easier to deal with responsiveness later 

```html 
    <div id="head-main-content">
      <h2>What’s on Spotify?</h2>

      <ul>
        <li>
          <img src="images/music-icon.png">
          <h3>Millions of Songs</h3>
          <p> There are millions of
            <br>songs on Spotify</p>
        </li>
        <li>
          <img src="images/high-quality-icon.png">
          <h3>HD Music</h3>
          <p>Listen to music as if you
            <br> were listening live</p>
        </li>
        <li>
          <img src="images/devices-icon.png">
          <h3>Stream Everywhere</h3>
          <p>Stream music on your
            <br>smartphone, tablet or
            <br> computer
          </p>
        </li>
      </ul>
    </div>
```